diff --git a/docusaurus.config.js b/docusaurus.config.js
index 712f36fd..80eb578c 100644
--- a/docusaurus.config.js
+++ b/docusaurus.config.js
@@ -17,7 +17,7 @@ const config = {
   url: "https://developers.iqm.com/",
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  baseUrl: "/",
+  baseUrl: "/api-docs/",
 
   trailingSlash: true,
 
